pkg/packagekit/updates.scss

```diff
@@ -153,6 +153,12 @@ div.changelog {
   }
 }
 
+.error-log {
+  font-size: var(--pf-global--FontSize--sm);
```

Comment:
> So I just tried and on mobile there is now an overflow, which makes this PR actually causing a regression for this page. The problem comes from the fact the Codeblock is using pre which does not allow lines to wrap and such.
> 
> Shall we better switch back to using text for the implementation? Why did you move to using codeblock anyway?
> 
> For seperating the lines we can just do:
> 
> ```
> +                                         {this.state.errorMessages
> +                                                 .filter((m, index) => index == 0 || m != this.state.errorMessages[index - 1])
> +                                                 .map(m => <Text key={m}>{m}</Text>)}
> ```
> 
> So using Text elements instead of span for the messages.

Using Text wont help. Will have to use TextVariant pre or code but will be getting the same overflow issue. The overflow issue can be solved with `whitespace: pre-wrap`. Is there a PF component for that?

Reply:
The overflow issue would be solved if we keep error block outside EmptyStatePanel's paragraph

Reply:
I see. so this output is not in seperate entries or of errorMessages array, but in one. There is not, I guess you can put a custom css for that only then.

Reply:
@KKoukiou Should I move it outside the paragraph? Overflow is still there after applying custom CSS.

Reply:
@KKoukiou the default font-size for pre seemed too large for when the error log was large. Also I thought it looked better on small screens when I used the above font size. 

```suggestion
  font-size: var(--pf-global--FontSize--sm);
```

Should I keep this font size?

Reply:
> pre which does not allow lines to wrap and such

Yeah, you need pre-wrap for that, which (AFAIK), isn't available in PF. So I guess it does need a sprinkling of custom CSS.

Also, does `paragraph={}` actually wrap it in `<p>` tags on the page? If so, that's wrong. If not, then it's most ideal to use it there (if it's just where detail text goes). 

In other words `<p>` tags cannot have block-level tags inside it.

Details:

- P can contain phrasing, but is flow content
   - https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element
- PRE can contain phrasing, but is flow content
   - https://html.spec.whatwg.org/multipage/grouping-content.html#the-pre-element
- phrasing cannot have flow content descendants
   - https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2

(Note: I'm not sure if PF's `paragraph` in that widget really renders to a paragraph tag `<p>` or not. Figuring this out — if it is rendered to `<p>` in the DOM — might help make a decision.)